### DISCUSSION TITLE
Vim is auto-documented, does Helix has this feature ? I can't find any doc about that.

### DISCUSSION THREAD

**Discussion by bouillerot** [helix-editor/helix#8463] [Q&A] [answered]
Well, the title seems to be full meaning. 

Is there, or will be there, a command as  ```:Help``` ?

Thanks by advance for any advice.

**Comment by the-mikedavis** [comment]
There isn't currently a `:help` command, see https://github.com/helix-editor/helix/issues/668

**Comment by bouillerot** [comment, marked as answer]
Thank you.
